app/controllers/tasks_controller.rb

```diff
@@ -17,6 +17,16 @@ def index
     @tasks = Task.all
   end
 
+  def complete
+    @task = Task.find(params[:id])
+    @task.mark_completed!
```

Comment:
Can this only "fail" by raising an exception? Or do we need to verify that it's worked?